docs-src/requirements.txt

```diff
@@ -1,3 +1,3 @@
 pydata-sphinx-theme
 sphinx >= 1.8
-sphinxprettysearchresults
+sphinxprettysearchresults @ git+https://github.com/biosimulators/sphinx-pretty-searchresults.git
```

Comment:
add issue to sphinxprettysearchresults to add an import os (so we don't need the fork).